README.md

```diff
@@ -266,6 +266,10 @@ Base font-sizing for body copy.
 ```sass
 $nb-base-font-size: 1rem !default;
 ```
+The delimiter to use for responsive variations of classes (default `@`, but could be changed to i.e. `-bp-` to allow `composes:` as that doesn't work with `@` in class names)
```

Comment:
We should mention that this is for use with CSS-Modules.  If the consumer isn't using them then everything should compile down fine.